.gitignore

```diff
@@ -9,4 +9,5 @@ MANIFEST
 .coverage
 .idea/
 __pycache__
-netlib.egg-info/
\ No newline at end of file
+netlib.egg-info/
+pathod/
```

Comment:
It is - somehow Travis creates this directory - and then the git-status check fails.
See https://travis-ci.org/mitmproxy/netlib/jobs/64226073#L199

Maybe it would be safer to remove the new Travis checks? What do you think?